Resources/Prototypes/Traits/quirks.yml

```
@@ -12,7 +12,7 @@
   name: trait-lightweight-name
   description: trait-lightweight-desc
   category: Quirks
-  cost: -1
+  cost: 0 # Mono
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)